BrowserKit/Sources/Shared/UserAgent.swift

```diff
@@ -175,7 +172,7 @@ public struct UserAgentBuilder {
     public static func defaultDesktopUserAgent() -> UserAgentBuilder {
         return UserAgentBuilder(
             product: UserAgent.product,
-            systemInfo: "(Macintosh; Intel Mac OS X 10.15)",
+            systemInfo: "(Macintosh; Intel Mac OS X 10_15_7)",
```

Comment:
This was aligned with the rest to match the actual desktop Safari, not desktop Firefox values, to blend in better not leaving any kinks in fingerprinting surface to help distinguishing from the masses.